uint/src/uint.rs

```diff
@@ -1688,6 +1688,11 @@ macro_rules! construct_uint {
 			type Err = $crate::FromHexError;
 
 			fn from_str(value: &str) -> $crate::core_::result::Result<$name, Self::Err> {
+                let value = if value.starts_with("0x") {
```

Comment:
Indentation looks off here.

Reply:
Seems like a github bug. It doesn't show in the Full review tab, and cargo fmt doesn't catch it

![image](https://user-images.githubusercontent.com/17802178/103229151-22c27700-493b-11eb-90a3-878a1299990a.png)

![image](https://user-images.githubusercontent.com/17802178/103229172-2f46cf80-493b-11eb-85a6-fe4ede4ab89b.png)

Reply:
It looks like you're using spaces (all crates here use tabs, `\t`) and that's why it renders weird on github. Can you convert to tabs please?